packages/core/admin/admin/src/content-manager/pages/EditView/utils/createAttributesLayout.js

```diff
@@ -1,14 +1,18 @@
 import { get, isEmpty } from 'lodash';
-// TODO: refacto this file to avoid eslint issues
-/* eslint-disable no-restricted-syntax */
-/* eslint-disable no-unused-vars */
 
-const createAttributesLayout = (currentLayout, attributes) => {
+const createAttributesLayout = (currentContentTypeLayoutData) => {
```

Comment:
Thanks for fixing the eslint warnings here!

I would recommend shuffling the order a bit:

- Move the early return as early as possible
- Move the `getType` after the declaration of `attributes` (I guess there's some hoisting going on, but it seems weird to me to use it before declaration)

Reply:
Yes good point I didn't catch that and I also prefer to have the variable used after declaration. I'll update.